js/transfer-button.js

```diff
@@ -1,20 +1,67 @@
 'use strict'
 
 var output = document.getElementById('greeter-output');
-output.innerHTML = 'Click the button to check temperature in Fahrenheit scale.' + '<br><br>' + output.innerHTML;
-
+output.innerHTML = 'Click the red button to check temperature in Fahrenheit scale.' + '<br><br>';
+var foutput = document.getElementById('greeter-foutput');
+foutput.innerHTML = 'Click the blue button to check temperature in Celcius scale.' + '<br><br>';
 var button = document.getElementById('greeter-button');
+var fbutton = document.getElementById('greeter-fbutton');
 var temperature;
 var Ftemperature;
```

Comment:
name of variables should start with small character (big letter is reserved for constructor functions in JS)